WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -90,46 +108,63 @@
 - [ ] Verify that bookmarks from other devices on the chain show up on the mobile device after sync completes
 - [ ] Verify newly created bookmarks get synced to all devices on the Sync chain
 - [ ] Verify existing bookmarks before joining Sync chain also gets sync'd to all devices on the Sync chain
-- [ ] Verify history sync is disabled by default. Enabling it starts sync'ing history items as well (limited to 200 entries).
+- [ ] Verify history sync is disabled by default. Enabling it starts sync'ing history items as well (limited to 200 entries)
 - [ ] Verify sync works on an upgraded profile and new bookmarks added post-upgrade sync across devices on the chain
 - [ ] Verify you can create a standalone Sync chain with one device
+- [ ] Verify starting sync chain using Other Devices segmented tab. Ensured enabling `open tabs` will display tabs from other devices
+- [ ] Verify sharing pages with `Send To Your Devices`
+- [ ] Verify `Sync Internals` page data is populated when connected to sync chain
+- [ ] Verify when using `Delete Sync Account` will remove all devices connected to sync chain
 
 ## Playlist & Autoplay
 
 - [ ] Verify Playlist is enabled by default on a clean install
 - [ ] Verify when a site with audio/video is detected, `Playlist` button is shown in URL bar and a notification badge on the menu
 - [ ] Verify clicking the `Playlist` button adds audio/video to playlist and initiates offline play
-- [ ] Verify able to add audio/video to playlist via share menu
-- [ ] Verify able to add audio/video to playlist via long press context menu
-- [ ] Verify Autoplay is disabled by default
+- [ ] Verify adding audio/video to playlist via share menu
+- [ ] Verify adding audio/video to playlist via long press context menu
+- [ ] Verify Autoplay is enabled by default
 - [ ] Verify with Autoplay enabled, able to play audio/video automatically
 - [ ] Verify with background audio enabled, able to play audio/video with device locked or app in background
+- [ ] Verify Playlist media works with CarPlay on iOS devices
+- [ ] Verify playback gestures are working in Playlist
+- [ ] Verify adding shared folder to playlist using `https://playlist.brave.com/bsa-sample`. Verify all actions on shared folder are working
```

Comment:
Suggestion

`Verify adding shared folder to playlist using https://playlist.brave.com/bsa-sample` - > `Verify adding shared folder to playlist using https://playlist.brave.com/bsa-sample works`